src/test/basics/base_uint_test.cpp

```diff
@@ -68,15 +93,15 @@ struct base_uint_test : beast::unit_test::suite
         {
             BEAST_EXPECT(d == --t);
         }
```

Comment:
Sure! I see some test format duplication in these three blocks: 
https://github.com/mellery451/rippled/blob/1e3f3b23f61d9eabb54033f2bb12c245c075850d/src/test/basics/base_uint_test.cpp#L64-L76
https://github.com/mellery451/rippled/blob/1e3f3b23f61d9eabb54033f2bb12c245c075850d/src/test/basics/base_uint_test.cpp#L84-L95
https://github.com/mellery451/rippled/blob/1e3f3b23f61d9eabb54033f2bb12c245c075850d/src/test/basics/base_uint_test.cpp#L104-L116
They are testing different items, but the expects are very similar. 
I was wondering what your thoughts were about abstracting the duplicate test functionality into a separate function versus leaving it as is.